samples/snippets/events/setup/setup_cleanup.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import datetime
+import os
+import re
+import shlex
+import subprocess
+
+from google.api_core.client_options import ClientOptions
+from google.cloud import storage
+from google.cloud.retail import ProductDetail, PurgeUserEventsRequest, \
+    UserEvent, UserEventServiceClient, WriteUserEventRequest
+from google.cloud.retail_v2 import Product
+from google.protobuf.timestamp_pb2 import Timestamp
+
+project_number = os.getenv('PROJECT_NUMBER')
+endpoint = "retail.googleapis.com"
+default_catalog = "projects/{0}/locations/global/catalogs/default_catalog".format(
+    project_number)
+
+
+# get user events service client
+def get_user_events_service_client():
+    client_options = ClientOptions(endpoint)
+    return UserEventServiceClient(client_options=client_options)
+
+
+# get user event
+def get_user_event(visitor_id):
+    timestamp = Timestamp()
+    timestamp.seconds = int(datetime.datetime.now().timestamp())
+
+    product = Product()
+    product.id = 'test_id'
+
+    product_detail = ProductDetail()
+    product_detail.product = product
+    product_detail.quantity = 3
```

Comment:
nit: quantity should not be set for "detail-page-view" event.

Reply:
done